github/issues_timeline.go

```diff
@@ -117,8 +117,10 @@ type Timeline struct {
 	// An object containing rename details including 'from' and 'to' attributes.
 	// Only provided for 'renamed' events.
 	Rename *Rename `json:"rename,omitempty"`
-	// The ProjectCard object is part of the starfox-preview,
-	// and will only be included when you pass the custom media type.
+	// A 'project_card' object that includes the 'id', 'url', 'project_id',
```

Comment:
On second thought, let's delete this comment entirely and let it start by itself. We'll let the `ProjectCard` struct documentation suffice. Sorry about that.